.vscode/launch.template.json

```diff
@@ -5,6 +5,22 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Python: Run Test",
+      "type": "python",
+      "request": "launch",
+      "module": "pytest",
+      "args": [
```

Comment:
this is a great addition to the template
I wonder if you want to add ``--setup-show`` as well. ( i find it very handy for debugging)

Reply:
done!